Makefile

```diff
@@ -17,10 +17,10 @@ clean:
 distclean: clean
 	./rebar delete-deps
 
-testclean: clean
```

Comment:
Asked in the TS room and no one is using it. Rebar detects changes so a clean should not be necessary.  It can always be readded.